applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -4,3 +4,399 @@
+			 CONFIG_BLE_ACL_SLAVE_LATENCY, CONFIG_BLE_ACL_SUP_TIMEOUT)
+
+#define DEVICE_NAME_PEER "NRF5340_AUDIO"
+#define DEVICE_NAME_PEER_LEN (sizeof(DEVICE_NAME_PEER) - 1)
+static struct net_buf_pool *iso_tx_pools[] = { LISTIFY(CONFIG_BT_ISO_MAX_CHAN,
+						       NET_BUF_POOL_PTR_ITERATE,
+						       (,)) };
+
+static struct bt_audio_lc3_preset unicast_present_nrf5340 =
+	BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO;
+
+static void start_scan(void);
+
+static int unicast_group_create(struct bt_audio_stream *stream)
+{
+	int ret;
+
+	ret = bt_audio_unicast_group_create(stream, CONFIG_BT_ISO_MAX_CHAN, &unicast_group);
+	if (ret != 0) {
+		LOG_ERR("Unable to create unicast group: %d", ret);
+		return ret;
+	}
+
+	return 0;
+}
+
+static void stream_configured_cb(struct bt_audio_stream *stream,
+				 const struct bt_codec_qos_pref *pref)
+{
+	int ret;
+
+	ret = bt_audio_stream_qos(default_conn, unicast_group, &unicast_present_nrf5340.qos);
+	if (ret != 0) {
+		LOG_INF("Unable to setup QoS: %d", ret);
+	}
+}
+
+static void stream_qos_set_cb(struct bt_audio_stream *stream)
+{
+	int ret;
+
+	ret = bt_audio_stream_enable(stream, unicast_present_nrf5340.codec.meta,
+				     unicast_present_nrf5340.codec.meta_count);
+	if (ret != 0) {
+		LOG_INF("Unable to enable stream: %d", ret);
+	}
+}
+
+static void stream_enabled_cb(struct bt_audio_stream *stream)
+{
+	int ret;
+
+	ret = bt_audio_stream_start(stream);
+	if (ret != 0) {
+		LOG_INF("Unable to start stream: %d", ret);
+	}
+}
+
+static void stream_started_cb(struct bt_audio_stream *stream)
+{
+	int ret;
+	struct event_t event;
+
+	event.event_source = EVT_SRC_LE_AUDIO;
+	event.le_audio_activity.le_audio_evt_type = LE_AUDIO_EVT_STREAMING;
+
+	ret = ctrl_events_put(&event);
+	ERR_CHK(ret);
+}
+
+static void stream_metadata_updated_cb(struct bt_audio_stream *stream)
+{
+	LOG_INF("Audio Stream %p metadata updated", (void *)stream);
+}
+
+static void stream_disabled_cb(struct bt_audio_stream *stream)
+{
+	LOG_INF("Audio Stream %p disabled", (void *)stream);
+}
+
+static void stream_stopped_cb(struct bt_audio_stream *stream)
+{
+	LOG_INF("Audio Stream %p stopped", (void *)stream);
+	int ret;
+	struct event_t event;
+
+	event.event_source = EVT_SRC_LE_AUDIO;
+	event.le_audio_activity.le_audio_evt_type = LE_AUDIO_EVT_NOT_STREAMING;
+
+	ret = ctrl_events_put(&event);
+}
+
+static void stream_released_cb(struct bt_audio_stream *stream)
+{
+	LOG_INF("Audio Stream %p released", (void *)stream);
+}
+
+static struct bt_audio_stream_ops stream_ops = {
+	.configured = stream_configured_cb,
+	.qos_set = stream_qos_set_cb,
+	.enabled = stream_enabled_cb,
+	.started = stream_started_cb,
+	.metadata_updated = stream_metadata_updated_cb,
+	.disabled = stream_disabled_cb,
+	.stopped = stream_stopped_cb,
+	.released = stream_released_cb,
+};
+
+static void add_remote_sink(struct bt_audio_ep *ep, uint8_t index)
+{
+	LOG_INF("Sink #%u: ep %p", index, (void *)ep);
```

Comment:
Hard to understand what this log means

Reply:
I also have concern in here, but when I was developing dual links example, this provides good help on tracking the process on different link.
But let remove this now.